components/vc/shared/wheelSpeed/wheelSpeed.c

```diff
@@ -34,8 +36,12 @@ typedef struct
 /******************************************************************************
  *                         P R I V A T E  V A R S
  ******************************************************************************/
+static struct
+{
+    float32_t    slipRatio;
+    wheelSpeed_S wheels;
+} wheelSpeed_data;
```

Comment:
If were going to integrate slip here, we should probably also first
A. Track axle speed (F/R). Probably good to have over CAN anyways
B. Calculate slip F/R slip as V_rear/V_front
C. Also maybe calculate F/R L/R wheel slip? I cant think of why diagonal slip would be important but @rickyelopez may have thoughts